root/user/profile.tt

```diff
@@ -91,6 +91,12 @@
             [% END %]
         [% END %]
 
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
+             WRAPPER property name=l('Last password login:');
```

Comment:
"password" part seems unnecessary since the only way to login is with a password. :)